chromadb/utils/embedding_functions.py

```diff
@@ -742,27 +742,54 @@ def __init__(
         """
 
         self._model_name = model_name
+        self._model_provider = self._model_name.split('.')[0]
+        
+        self._bedrock_client = session.client(
+            service_name="bedrock",
+            **kwargs
+        )
+        
+        self._model_details = self._bedrock_client.get_foundation_model(modelIdentifier=self._model_name)['modelDetails']
+        assert "EMBEDDING" in self._model_details['outputModalities'], f"{self._model_name} doesn't have embedding modality output!"
```

Comment:
Do you mind using the `if` statement here? Assertions can be turned off, so this check won't pass.

Reply:
Old tester habit, will fix that in a sec